witchcraft/refreshable/refreshable_file_test.go

```diff
@@ -0,0 +1,155 @@
+// Copyright (c) 2018 Palantir Technologies. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package refreshable
+
+import (
+	"context"
+	"io/ioutil"
+	"os"
+	"path"
```

Comment:
Replace path with filepath so the test works for windows devs

Reply:
Done